R/learnr.R

```diff
@@ -0,0 +1,136 @@
+
+
+#' Learnr methods for sortable_learnr
+#'
+#' Questions will be created using the first set of options.
+#' Each set of answer options must contain the same set of answer options.
+#' When the question is completed, the first correct answer will be displayed.
+#'
+#' @param ... parameters passed onto \code{learnr::\link[learnr]{question}}.
+#' @template options
+#' @export
+#' @examples
+#' if (require(learnr, quietly = TRUE)) {
+#'   # to be used within a learnr tutorial...
+#'   question_sortable(
+#'     "Sort the first 5 letters",
+#'     answer(letters[1:5], correct = TRUE),
+#'     allow_retry = TRUE,
+#'     options = sortable_options(animation = 150)
+#'   )
+#' }
+question_sortable <- function(..., options = sortable_options()) {
```

Comment:
I think we should explicitly set `random_answer_order = TRUE` and pass this on to `learnr`.

Reply:
Added